src/caf/space/inputs.py

```diff
@@ -157,6 +161,59 @@ def _valid_data_col(cls, v, values):
         return v
 
 
+@dataclasses.dataclass
+class SpaceArguments:
+    """Command Line arguments for running space."""
+
+    config_path: Path
+    mode: str
```

Comment:
A custom Enum class would make this variable more obvious what it's possible values are.